Resources/Prototypes/_NF/Reagents/chemicals.yml

```
@@ -4,3 +4,33 @@
   desc: reagent-desc-salicylicacid
   physicalDesc: reagent-physical-desc-powdery
   color: "#EEEEEE"
+
+- type: reagent
+  id: JuiceThatMakesYouOOOK
+  name: reagent-name-juicethatmakesyouoook
+  desc: reagent-desc-juicethatmakesyouoook
+  physicalDesc: reagent-physical-desc-thick
+  color: "#7c3d00"
+  metabolisms:
+    Poison:
+      metabolismRate: 0.25
+      effects:
+        - !type:speechSounds
+          saySound: Monkey
+          showInChat: true
+          force: true
+          probability: 0.5
+        - !type:Polymorph
+          prototype: ArtifactMonkey
+          conditions:
+            - !type:OrganType
+              type: Animal
+              shouldHave: false
+            - !type:ReagentThreshold
+              min: 10
+        - !type:AdjustReagent
+          reagent: JuiceThatMakesYouOOOK
```

```suggestion
          reagent: JuiceThatMakesYouOok
```
Consistency if you change the other OOOK instances.